### DISCUSSION TITLE
project beta - how to delete?

### DISCUSSION THREAD

**Discussion by lohhonch** [community/community#7099] [Projects and Issues] [answered]
as titled, how can i delete/purge a project beta entirely?

**Comment by rampd2** [comment]
possible duplicate of #5436
I think it is better to upvote the already existing feedback post instead of creating many.

**Comment by 2percentsilk** [comment, marked as answer]
Thanks for your patience and feedback! We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need). Give it a try and let us know how it goes 😄
